packages/flutter/lib/src/material/banner.dart

```diff
@@ -120,6 +123,14 @@ class MaterialBanner extends StatefulWidget {
   /// Typically this is a list of [TextButton] widgets.
   final List<Widget> actions;
 
+  /// The z-coordinate at which to place the material banner. This controls the size of the shadow below the material banner.
+  ///
+  /// Defines the banner's [Material.elevation].
+  ///
+  /// If this property is null, then [MaterialBannerThemeData.elevation] of [ThemeData.bannerTheme] is used, if that is also null, the default value is 0.
+  /// If the elevation is 0, the Scaffold's body will be pushed down by the MaterialBanner.
```

Comment:
We generally keep documentation to 80 chars per line. Could you rework this to match that?